### PR TITLE
Fix DeltaEJz algorithm

### LINKED ISSUE #89: Bug in deltaEJz? yellows and blues strangly close together
I noticed that for the `deltaEJz` algorithm, the yellows and blues are strangely close together. E.g., `goldenrod` is apparently closer to blues like `cornflowerblue` or `royalblue` than to `peru` or `khaki`? That seems ... unintuitive.

The other color functions seem to agree that the yellows should be much closer to each other than to the blues.

For your convenience, I have created this comparison of color distances between yellows and blues and different distance functions. It works really nicely when copied into one of your incredibly useful notebooks:

```js
let goldenrod = new Color("goldenrod");

let cornflowerblue = new Color("cornflowerblue");
let royalblue = new Color("royalblue");
let slateblue = new Color("slateblue");
let peru = new Color("peru");
let khaki = new Color("khaki");
let darkorange = new Color("darkorange");


goldenrod.deltaEJz(cornflowerblue);
goldenrod.deltaEJz(royalblue);
goldenrod.deltaEJz(slateblue);
goldenrod.deltaEJz(peru);
goldenrod.deltaEJz(khaki);
goldenrod.deltaEJz(darkorange);

goldenrod.deltaE76(cornflowerblue);
goldenrod.deltaE76(royalblue);
goldenrod.deltaE76(slateblue);
goldenrod.deltaE76(peru);
goldenrod.deltaE76(khaki);
goldenrod.deltaE76(darkorange);

goldenrod.deltaE2000(cornflowerblue);
goldenrod.deltaE2000(royalblue);
goldenrod.deltaE2000(slateblue);
goldenrod.deltaE2000(peru);
goldenrod.deltaE2000(khaki);
goldenrod.deltaE2000(darkorange);

goldenrod.deltaECMC(cornflowerblue);
goldenrod.deltaECMC(royalblue);
goldenrod.deltaECMC(slateblue);
goldenrod.deltaECMC(peru);
goldenrod.deltaECMC(khaki);
goldenrod.deltaECMC(darkorange);

goldenrod.deltaEITP(cornflowerblue);
goldenrod.deltaEITP(royalblue);
goldenrod.deltaEITP(slateblue);
goldenrod.deltaEITP(peru);
goldenrod.deltaEITP(khaki);
goldenrod.deltaEITP(darkorange);
```

However, I really don't have a clue about colors at all, let alone color space and not to mention distance functions in those spaces. So it might very well be that this is the way it is supposed to be for that `deltaEJz` distance function, in which case I'm very sorry for the noise 😔.

### PR DESCRIPTION
**Pull request by facelessuser**
Fixes #89

### CODE REVIEW

File: src/deltaE/deltaEJz.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ Color.prototype.deltaEJz = function (sample) {
 	}
 
 	let Δh = Hz1 - Hz2;
-	let ΔH = 2 * Math.sqrt(Cz1 * Cz2) * Math.sin(Δh * Math.PI / 180);
+	let ΔH = 2 * Math.sqrt(Cz1 * Cz2) * Math.sin((Δh / 2) * (Math.PI / 180));
```
Comment (svgeesus) [inline src/deltaE/deltaEJz.js:42]: The argument to Math.sin is in radians.

The original code converts degrees to radians (but fails to divide by two)
The new code passes the value in degrees and then multiplies the result by Π/180 !

We actually want 

```js
Math.sin((Δh / 2) * (Math.PI / 180));

**Review by svgeesus — CHANGES_REQUESTED** [review]
See comment

File: src/deltaE/deltaEJz.js (hunk 1 above)
Comment (facelessuser) [reply src/deltaE/deltaEJz.js:42]: Cool, I'll take a look today.

Comment (facelessuser) [reply src/deltaE/deltaEJz.js:42]: @svgeesus What I provided in this pull does not pass the value by degrees. It is actually identical to what you are suggesting. I suspect maybe a trick of the eye when looking at the parenthesis in the delta?

Comment (svgeesus) [reply src/deltaE/deltaEJz.js:42]: Yes, you are right, my bad

**Review by svgeesus — APPROVED** [review]

**Pull request merged**